### PR TITLE
Dependency updates August 24 (2.2.0)

### PR DESCRIPTION
**Pull request by MyPyDavid**
## Description

collects new dependency updates for merge in `dev-2.2.0`

## Types of Changes

- [x] Build related changes

## Checklist

- [ ] All new and existing tests passed.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -76,14 +76,14 @@ dev = [
...
+  "setuptools>=69,<73",
   "twine~=5.1.1",
-  "wheel>=0.42,<0.44",
+  "wheel>=0.42,<0.45",
   "rdmo[allauth]",
   "rdmo[pytest]",
 ]
 gunicorn = [
-  "gunicorn>=21.2,<23.0",
+  "gunicorn>=21.2,<24.0",
```
Comment (MyPyDavid) [inline pyproject.toml:86]: do we need to bump the minimum of the `gunicorn` (https://github.com/benoitc/gunicorn/releases) version here @jochenklar?  
I am out of clue for this

Comment (jochenklar) [reply pyproject.toml:86]: yes, go for it (gunicorn is pretty stable I guess, I don't expect problems).

File: pyproject.toml [hunk 2]
```diff
@@ -66,7 +66,7 @@ dependencies = [
 
 [project.optional-dependencies]
 allauth = [
-  "django-allauth>=0.61.1,<0.64.0",
+  "django-allauth>=0.61.1,<64.2.0",
```
Comment (MyPyDavid) [inline pyproject.toml:69]: same holds for allauth, should I bump the minimum version? https://docs.allauth.org/en/latest/release-notes/recent.html

Comment (jochenklar) [reply pyproject.toml:69]: ok with me.

Comment (MyPyDavid) [reply pyproject.toml:69]: they have split up the dependencies and our typical use-cases are the `socialaccounts` or OpenID providers, should I add it like this?
ref: https://docs.allauth.org/en/latest/socialaccount/providers/openid.html#openid
```
django-allauth[socialaccount,openid]>=64.1.0,<64.2.0",
```

Comment (jochenklar) [reply pyproject.toml:69]: Yes, I don't think we need the others: https://github.com/pennersr/django-allauth/blob/main/setup.cfg#L50 (steam anyone?)

Comment (MyPyDavid) [reply pyproject.toml:69]: `saml` could be interesting, but I haven't tested it so far, it could simplify the shibboleth setup.
`mfa`, multi-factor?

Comment (jochenklar) [reply pyproject.toml:69]: yes, `saml` could be useful, but I don't want to enter this business. Feel free to try that with your setup if you find the time. `mfa` is something for the future I guess. People can always install more stuff and costimize rdmo locally.

**Review by jochenklar — APPROVED** [review]

**Comment by jochenklar** [comment]
Jo, merge away.

**Pull request merged**